### DISCUSSION TITLE
More Browsers

### DISCUSSION THREAD

**Discussion by TheVioletSpiral** [ScratchAddons/ScratchAddons#5921] [Ideas] [answered]
scratch addons is great, but the only browsers it supports are Chrome, which is slow for some reason; Fire Fox, which is actually just fine; and Edge, which is horrible. I really hope the person or people behind scratch addons will make it available for more browsers. I would really like one for OperaGX, because it's my favorite browser.

**Comment by Samq64** [comment]
The Chrome version should run on (almost) anything except Firefox or Safari.

**Comment by WorldLanguages** [comment, marked as answer]
OperaGX is a Chromium fork and can download extensions from the Chrome Web Store.

**Reply by TheVioletSpiral** [reply]
i'll try that thanks!

**Comment by ilikecereal1legacy** [comment]
how about safari?

**Reply by Samq64** [reply]
See #340 and #3452

**Comment by Secret-chest** [comment]
> how about safari?

No way we can do that, search for an issue there.
